src/src/StackBuffer.hpp

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
So the StackArray is just a regular array with a StackBuffer now?

Reply:
Yup. It's the same as before but with a `StackBuffer` instead of a `StackArrayWrapper`.